src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html

```diff
@@ -5,19 +5,21 @@
     {% with organization_page=organization.extended_object %}
         <a
           href="{{ organization_page.get_absolute_url }}"
-          title="{{ organization_page.get_title }}"
+          title="{% trans "Main organization" %} &quot;{{ organization_page.get_title }}&quot;"
+          aria-label="{% trans "Main organization" %} &quot;{{ organization_page.get_title }}&quot;"
           class="subheader__cartouche"
           property="{{ organization_property|default:'provider' }}"
           typeof="CollegeOrUniversity"
         >
             <meta property="name" content="{{ organization_page.get_title }}"/>
             <meta property="url" content="{{ SITE.web_url }}{{ organization_page.get_absolute_url }}"/>
             {% get_placeholder_plugins "logo" organization_page as plugins or %}
-                <div class="subheader__empty">{% trans "Main organization" %}</div>
+                <div class="subheader__empty" aria-hidden="true">{% trans "Main organization" %}</div>
             {% endget_placeholder_plugins %}
             {% blockplugin plugins.0 %}
                 <div class="subheader__media">
                     <img
+                      alt=""
```

Comment:
the alt here is now empty, as the text line 17 is hidden, because the text is already in the `title` and `aria-label` on the wrapping `<a>`